CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
   - Add support to decompile jar as a java project when finding the definition of a java class, allowing to have LSP features on that java project. #1187
   - Add refactorings similar to `Sort map`: `Sort vector`, `Sort list`, `Sort set`, and `Sort clauses` for functions like `assoc`. #1155
 
+- Build
+  - Replace Makefile and MS-Windows scripts with bb tasks. #1211
```

Comment:
Usually, we add an entry to refer to changes that affect users, this is not one that will affect the final user so not sure it's worth polluting the changelog with these changes, WDYT?

Reply:
Done, hopefully existing developers won't have much trouble figuring out the move to bb :)